web-src/src/admin/components/scripts-config/ScriptConfig.vue

```diff
@@ -8,6 +8,9 @@
           <h5>Parameters</h5>
           <ScriptParamList :parameters="scriptConfig.parameters"/>
         </div>
+        <div v-if="scriptName !== NEW_SCRIPT">
```

Comment:
Should we put it in the footer? Where Save button resides

Reply:
It might be more fitting, because the rest belongs to the script, and the save/create/delete are just for storing them

Reply:
hmm... I'm not sure to style this properly in this case 😅
![image](https://user-images.githubusercontent.com/48969167/229111113-041202fd-f557-40dd-881b-0875f167e4ad.png)

Reply:
I pushed a commit with moving and styling this button
FYI I removed the background, to avoid extra attention to this button on UI + in order to avoid having static colors (because they don't play nicely with dark theme support)